apps/api/src/app/organization/usecases/update-organization/update-organization-command.ts

```diff
@@ -0,0 +1,9 @@
+import { IsDefined } from 'class-validator';
+import { AuthenticatedCommand } from '../../../shared/commands/authenticated.command';
+
+export class UpdateOrganizationCommand extends AuthenticatedCommand {
+  @IsDefined()
+  public readonly id: string;
+
+  name: string;
```

Comment:
@Abobos due you think we should add here some maxLen for the organization name? Would use something permissive here, but maybe providing some limit will be a good idea to not break UI and etc...

Reply:
Could you throw more perspective?.. How would this break this UI if we don't limit it?

Reply:
For example everywhere we show the organization name it can overflow (the settings dropdown, organization select) and maybe other places. This is quite and edge case to be honest. Really not a must.

Reply:
@scopsy yes. this makes sense.  done